src/treetensornetworks/opsum_to_ttn.jl

```diff
@@ -13,21 +13,13 @@ using StaticArrays: MVector
 # Utility methods
 # 
 
-# linear ordering of vertices in tree graph relative to chosen root, chosen outward from root
-function find_index_in_tree(site, g::AbstractGraph, root_vertex)
-  ordering = reverse(post_order_dfs_vertices(g, root_vertex))
-  return findfirst(x -> x == site, ordering)
-end
-function find_index_in_tree(o::Op, g::AbstractGraph, root_vertex)
-  return find_index_in_tree(ITensors.site(o), g, root_vertex)
-end
-
 # determine 'support' of product operator on tree graph
 function span(t::Scaled{C,Prod{Op}}, g::AbstractGraph) where {C}
-  spn = eltype(g)[]
+  spn = Set{eltype(g)}()
   nterms = length(t)
-  for i in 1:nterms, j in i:nterms
-    path = vertex_path(g, ITensors.site(t[i]), ITensors.site(t[j]))
+  nterms == 1 && return Set([ITensors.site(t[1])])
+  for i in 1:nterms, j in (i + 1):nterms
+    path = Set(vertex_path(g, ITensors.site(t[i]), ITensors.site(t[j])))
     spn = union(spn, path)
```

Comment:
Seems like this could be `union!`, not sure if it makes a performance difference in practice though.